cmd/osm-controller/osm-controller.go

```diff
@@ -215,7 +215,7 @@ func main() {
 	}
 
 	// Create and start the ADS gRPC service
-	xdsServer := ads.NewADSServer(meshCatalog, cfg.IsDebugServerEnabled(), osmNamespace, cfg, certManager)
+	xdsServer := ads.NewADSServer(meshCatalog, cfg.IsDebugServerEnabled(), osmNamespace, cfg, certManager, cfg.GetMaxDataPlaneConnections())
```

Comment:
No need to pass this around, use it directly in `stream.go`

Reply:
Whoops, you're right, will go ahead and make the changes!